libraries/utils/src/shared-components/drop-down.ts

```diff
@@ -1,66 +1,91 @@
 import * as ui from 'datagrok-api/ui';
+import {Observable, fromEvent} from 'rxjs';
+import {filter} from 'rxjs/operators';
+
+import '../../css/drop-down.css';
+
 
 export class DropDown {
-  private _elements: string[];
-  private _root?: HTMLDivElement;
-  private _dropDownElements?: HTMLDivElement;
+  private _element: HTMLElement;
+  private _dropDownElement: HTMLDivElement;
   private _isMouseOverElement: boolean;
-  private _expanded: boolean;
+  private _label: string | Element;
+
+  isExpanded: boolean;
+  root: HTMLDivElement;
 
-  constructor(elements: string[]) {
-    this._elements = [];
+
+  constructor(label: string | Element, elementCreation: () => HTMLElement) {
```

Comment:
Typically, verbs are used in function names (`getElement` or `createElement` instead of `elementCreation`)